src/fides/api/schemas/connection_configuration/connection_secrets_mysql.py

```diff
@@ -40,6 +40,12 @@ class MySQLSchema(ConnectionConfigSecretsSchema):
         title="SSH required",
         description="Indicates whether an SSH tunnel is required for the connection. Enable this option if your MySQL server is behind a firewall and requires SSH tunneling for remote connections.",
     )
+    sslmode: Optional[str] = Field(
+        None,  # TODO: support for verify-ca and verify-full
+        title="SSL Mode",
+        description="The SSL mode to use for the connection. Valid values are 'required', 'preferred', and 'disabled'.",
+        pattern=r"required|preferred|disabled",
+    )
```

Comment:
We could use an enum for this
```
class SSLMode(str, Enum):
    preferred = "preferred"
    required = "required"
    disabled = "disabled"
```
Then we could just do this
```suggestion
    sslmode: Optional[SSLMode] = Field(
        None,  # TODO: support for verify-ca and verify-full
        title="SSL Mode",
        description="The SSL mode to use for the connection. Valid values are 'required', 'preferred', and 'disabled'."
    )
```

Reply:
@galvana could I define that ENUM in `connection_secrets_mysql.py`?